include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -4665,7 +4671,128 @@ struct OrtApi {
                   _In_reads_(num_external_initializer_files) char* const* external_initializer_file_buffer_array,
                   _In_reads_(num_external_initializer_files) const size_t* external_initializer_file_lengths,
                   size_t num_external_initializer_files);
-};
+
+  /** \brief Create OrtDevice object.
```

Comment:
>/** \brief Create OrtDevice object. [](http://example.com/codeflow?start=2&length=35)

Suggestion is to create a table that is separate from
the main API. Reasons:
Most of the clients do not need that code.
But it does affect language bindings. 
For example, we now need to pad C#  imported API structure, although it is unlikely we would ever need that in the C#, but if we do we can add that separate.